### PR TITLE
KotlinCleanup: AbstractFlashcardViewer

### PR DESCRIPTION
**Pull request by sudhanshu1309**
Kotlin Cleanup in AbstractFlashcardViewer

## Fixes
Fixes a part of #10489 

## How Has This Been Tested?
Unit tests

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 1]
```diff
@@ -446,7 +446,6 @@ abstract class AbstractFlashcardViewer :
             dealWithTimeBox()
         }
 
-        @KotlinCleanup("remove _ variables")
```
Comment (sudhanshu1309) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:449]: _ s had already been removed. Didn't found any one.

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 2]
```diff
@@ -659,11 +657,8 @@ abstract class AbstractFlashcardViewer :
         } else super.onKeyDown(keyCode, event)
     }
 
-    @KotlinCleanup("Use ?:")
```
Comment (sudhanshu1309) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:662]: better to use ?. not ?:

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 3]
```diff
@@ -1216,20 +1204,19 @@ abstract class AbstractFlashcardViewer :
...
     }
 
-    @KotlinCleanup("collapse _ variables")
     protected open fun initControls() {
         mCardFrame!!.visibility = View.VISIBLE
         mPreviousAnswerIndicator!!.setVisibility(View.VISIBLE)
         flipCardLayout!!.visibility = View.VISIBLE
         answerField!!.visibility = if (typeAnswer!!.validForEditText()) View.VISIBLE else View.GONE
-        answerField!!.setOnEditorActionListener { _: TextView?, actionId: Int, _: KeyEvent? ->
+        answerField!!.setOnEditorActionListener { _, actionId: Int, _ ->
```
Comment (sudhanshu1309) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:1212]: @KotlinCleanup("collapse _ variables") meant this or something else?

Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:1212]: The changes are ok.

**Review by lukstbit — APPROVED** [review]
Thank you for contributing!

**Review by BrayanDSO — APPROVED** [review]

**Pull request merged**